src/PhotoNim/shapes.nim

```diff
@@ -143,4 +143,17 @@ method fastIntersection*(plane: Plane, ray: Ray): bool =
     let t = -inv_ray.start.z / inv_ray.dir[2]
     if t < inv_ray.tmin or t > inv_ray.tmax: return false
     
-    return true
\ No newline at end of file
+    return true
+
+proc add*(scenary: var World, shape: Shape) {.inline} = scenary.shapes.add(shape)
+    ## Procedure to add a shape to a sequence of shapes
+
+proc get*(scenary: World, ind: int): Shape {.inline.} = scenary.shapes[ind]
+    ## Procedure to get a shape which is a part of the scenary
+
```

Comment:
I find add and get procs not so useful: you can access the shapes with scenary.shapes

Reply:
I would remove these procs.